main.py

```diff
@@ -156,62 +156,66 @@ async def get_settings(ctx: discord.ApplicationContext):
 
 @bot.event
 async def on_message( message: discord.Message):
-    if message.author == bot.user: return #if the message is sent by the bot, we don't want to moderate it, we don't want to moderate the bot,
-    try: c.execute("SELECT * FROM moderation WHERE guild_id = ?", (str(message.guild.id),))
-    except: return
-    data = c.fetchone()
-    try: c.execute("SELECT * FROM data WHERE guild_id = ?", (str(message.guild.id),))
-    except: return
-    data2 = c.fetchone()
-    if data is None: return
-    channel = message.guild.get_channel(int(data2[1]))
-    is_enabled = data2[2]
-    moderator_role_id = data2[3]
-    content = message.content
+  unbanned_words = ["kick", "kicks", "kicking"]
+  for word in unbanned_words:
+        if word in message.content:
+            return
+        if message.author == bot.user: return #if the message is sent by the bot, we don't want to moderate it, we don't want to moderate the bot,
+        try: c.execute("SELECT * FROM moderation WHERE guild_id = ?", (str(message.guild.id),))
+        except: return
+        data = c.fetchone()
+        try: c.execute("SELECT * FROM data WHERE guild_id = ?", (str(message.guild.id),))
+        except: return
+        data2 = c.fetchone()
+        if data is None: return
+        channel = message.guild.get_channel(int(data2[1]))
+        is_enabled = data2[2]
+        moderator_role_id = data2[3]
+        content = message.content
```

Comment:
This whole piece of code does _not_ need to be run for **every** unbanned word. This should just be ran once to setup the guild and other properties. This will be super bad for performance when the whitelist grows